src/shared/sleep-config.c

```diff
@@ -802,6 +802,8 @@ static bool location_is_resume_device(const HibernateLocation *location, dev_t s
  * Attempt to find the hibernation location by parsing /proc/swaps, /sys/power/resume, and
  * /sys/power/resume_offset.
  *
+ * Beware: For security reasons, never find this by scanning for inactive swap devices!
```

Comment:
Could you also explain why here? Also adding the same comment in the commit message should be fine.

Reply:
Sure. I thought you guys would rather want it short, but I actually also prefer an elaborate description.